lib/reek/rake/task.rb

```diff
@@ -32,118 +33,87 @@ module Rake
     #   rake reek REEK_OPTS=-s                   # sorts the report by smell
     #
     class Task < ::Rake::TaskLib
-      # Name of reek task.
-      # Defaults to :reek.
-      attr_accessor :name
+      # Name of reek task. Defaults to :reek.
+      attr_writer :name
 
-      # Array of directories to be added to $LOAD_PATH before running reek.
-      # Defaults to ['<the absolute path to reek's lib directory>']
-      attr_accessor :libs
-
-      # Glob pattern to match config files.
+      # Path to reeks config file.
       # Setting the REEK_CFG environment variable overrides this.
-      # Defaults to 'config/**/*.reek'.
-      attr_accessor :config_file
-
-      def config_files=(value)
-        @config_file = value
-      end
-
-      def config_files
-        @config_file
-      end
+      attr_writer :config_file
```

Comment:
BTW we could easily get rid of this as well since there's absolutely nothing keeping you from putting this into REEK_OPTS. I left it in to not f**k up everybodies rake tasks.